ee-9/feature-pack/src/main/resources/license/preview-feature-pack-licenses.xml

```diff
@@ -524,8 +524,8 @@
       </licenses>
     </dependency>
     <dependency>
-      <groupId>org.jboss.spec.javax.xml.ws</groupId>
-      <artifactId>jboss-jaxws-api_2.3_spec</artifactId>
+      <groupId>org.jboss.spec.jakarta.xml.ws</groupId>
+      <artifactId>jboss-jakarta-xml-ws-api_3.0_spec</artifactId>
```

Comment:
Please move this up into the right alphabetical order spot based on the new GA.

Reply:
Fixed.